python/pyspark/pandas/generic.py

```diff
@@ -3259,6 +3260,10 @@ def interpolate(
             Maximum number of consecutive NaNs to fill. Must be greater than
             0.
 
+        limit_direction : str, default None
+            Consecutive NaNs will be filled in this direction.
+            One of {{'forward', 'backward', 'both'}}.
```

Comment:
Could we elaborate some more detail as pandas does ?

<img width="587" alt="Screen Shot 2022-04-19 at 11 34 35 AM" src="https://user-images.githubusercontent.com/44108233/163908818-d6ee8883-8685-402d-8cbc-f983f5565c13.png">

Reply:
since `ffill`/`bfill` is not supported in `interpolate` for now, so I don't add those descriptions.

`ffill`/`bfill` is now only supported in `fillna`.

Reply:
Got it!